### PR TITLE
Fix music player

### PR DESCRIPTION
**Pull request by drakeerv**

### CODE REVIEW

File: community/src/music.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import pyglet.media
+
+class MusicPlayer(pyglet.media.Player):
+    def __init__(self):
+        super().__init__()
+        
+        self.standby = False
+        self.next_time = 0
```
Comment (obiwac) [inline community/src/music.py:8]: This file uses spaces and is missing a newline at the end.

Could you run the formatter on it?

It's weird that the GitHub action isn't running on this branch...

**Review by obiwac — CHANGES_REQUESTED** [review]

**Comment by obiwac** [comment]
I have rebased `community-restructuring` so you're going to have to rebase your branch too before merging

File: community/src/music.py (hunk 1 above)
Comment (drakeerv) [reply community/src/music.py:8]: I thought my vscode was configured properly... I'll fix it when I get the chance

**Review by obiwac — APPROVED** [review]

**Comment by obiwac** [comment]
I have pulled in your changes directly as I don't think I can rebase someone else's PR.

**Pull request closed without merging**